models/postgrest/filteroperandtype_string.go

```diff
@@ -0,0 +1,25 @@
+// Code generated by "stringer -type=FilterOperandType"; DO NOT EDIT.
```

Comment:
What is this for?

Reply:
Auto-geneated `FilterOperandType.String()` method, generated by `go generate`: https://github.com/scratchdata/ScratchDB/blob/d7ff953659eba70350ab6f8fb53f08a862167161/models/postgrest/postgrest.go#L1